.github/workflows/coverage.yml

```diff
@@ -73,5 +65,4 @@ jobs:
         uses: coverallsapp/github-action@v2.2.1
         with:
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          path-to-lcov: coverage.info
```

Comment:
Maybe temporarily enable coverage for this PR because I've tried that and it doesn't work

Reply:
You can also try in my PR https://github.com/iotaledger/iota-sdk/pull/1035